.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+[Jira ticket](https://dsdmoj.atlassian.net/browse/EL-XXX)
+
+## What changed and Why?
+
+<!-- EXAMPLE, fill this in
+
+I have added/removed/altered:
+
+- [ ] Added foo in bar
+- [ ] Removed baz
+
+I am doing this because:
+
+-
+-
+- 
+
+-->
+
+## Guidance to review (optional)
+
+<!-- Anything useful to let the reviewer know? 
+
+-
+
+-->
+
+## Checklist
+
+Before you ask people to review this PR:
+
+- Tests and linting should be passing.
```

Comment:
I think for this section we can create checkboxes for our reviewers to check off as they go along here. I've also slightly shortened your steps. What do you think, let me know.

```
- [ ] **Tests and linting** are passing.  
- [ ] **Branch is up to date** with `main` (no merge conflicts).  
- [ ] **No unnecessary whitespace changes** (avoid unnecessary diffs).  
- [ ] **PR description clearly explains** what changed and why, with a JIRA/Trello link.  
- [ ] **Diff has been checked** for any unexpected changes.  
- [ ] **Commit messages are clear** and explain individual changes properly.  
```

Reply:
have ammeneded